cmp/grid/impl/GridPersistenceModel.js

```diff
@@ -86,7 +85,7 @@ export class GridPersistenceModel extends HoistModel {
         return {
             track: () => this.gridModel.columnState,
             run: (columnState) => {
-                this.patchState({columns: this.cleanColumnState(columnState)});
+                this.patchState({columns: this.gridModel.cleanColumnState(columnState)});
```

Comment:
why does this columnState have to be cleaned again if it is coming directly out of gridModel.  Isn't that by definition clean?

Reply:
Probably to remove widths? IE:

// Remove the width from any non-resizable column - we don't want to track those widths as
// they are set programmatically (e.g. fixed / action columns), and saved state should not
// conflict with any code-level updates to their widths.

Reply:
can we have the method on the persistenceModel renamed to removeWidths and only do that?  Confusing to have the duplication of the other code.